server/Player.js

```diff
@@ -181,10 +189,28 @@ class Player {
     }
   }
 
+  energyUse(type) {
+    if (type === "attack" && this.energy >= ENERGY_ATTACK_USE) {
+      this.energy -= ENERGY_ATTACK_USE;
+    }
+  }
+
+  energyRegenerate() {
+    if (!this.isDead && this.energy < ENERGY_MAX) {
```

Comment:
What if you will have 98% of energy? Then you will never reach the 100.
Maybe do it similarly to this
https://github.com/neu5-przenz-games/the-game/blob/905605fc3b393d206477aab5c08bd47aba5d0995/server/Player.js#L135
Just add whatever there is and if it's more than 100 make it 100.

Reply:
I'm not sure what's the problem, please check the code again.